integreat_cms/cms/templates/events/event_form.html

```diff
@@ -89,15 +89,17 @@ <h1 class="heading overflow-hidden text-ellipsis">
                             <a id="slug-link"
                                href="{{ url_link }}{{ event_translation_form.instance.slug }}"
                                class="text-blue-500 hover:underline">{{ url_link }}{{ event_translation_form.instance.slug }}</a>
-                            <a id="edit-slug-btn"
-                               title="{% translate "Edit" %}"
-                               class="mx-2 btn-icon">
-                                <i icon-name="edit-3"></i>
-                            </a>
+                            {% if can_change_event %}
+                                <a id="edit-slug-btn"
+                                   title="{% translate "Edit" %}"
+                                   class="mx-2 btn-icon">
+                                    <i icon-name="edit-3"></i>
+                                </a>
+                            {% endif %}
                             <a id="copy-slug-btn"
                                data-copy-to-clipboard="{{ event_translation_form.instance.base_link }}{{ event_translation_form.instance.slug }}"
                                title="{% translate "Copy to clipboard" %}"
-                               class="btn-icon">
+                               class="{% if not can_change_event %} ml-2{% endif %} btn-icon">
```

Comment:
Instead of having some buttons with the `mx-2` class and some buttons without margin at all, I think it would make more sense to assign a consistent spacing of `ml-2` or `mr-2` to every single one of the buttons. This way it does not influence the other buttons when one button is changed.
The existing margins are not ideal in my opinion, so now would be a good opportunity to fix them.

Reply:
That's a good idea 😃 
I've made changes so each button in form has ml-2.